cli/ocspsign/ocspsign.go

```diff
@@ -71,5 +71,16 @@ func ocspSignerMain(args []string, c cli.Config) (err error) {
 	return
 }
 
+// SignerFromConfig creates a signer from a cli.Config as a helper for cli and serve
+func SignerFromConfig(c cli.Config) (ocsp.Signer, error) {
+	//if this is called from serve then we need to use the specific responder key file
+	//fallback to key for backwards-compatibility
+	k := c.ResponderKeyFile
```

Comment:
Ya I don't disagree but I didnt want to break backwards compatibility. Can we put a depreciation warning and a todo to remove it?